docs/index.rst

```diff
@@ -40,6 +40,13 @@ of the ``curl`` call, like this::
    dns/rrsets
 
 
+.. toctree::
+   :maxdepth: 2
+   :caption: Applications
+
+   applications/lets-encrypt
```

Comment:
`Integrations` would be much better

Reply:
... and I would move it below dynDNS (before API Summary)

Reply:
done